tiledb/tests/test_attribute.py

```diff
@@ -0,0 +1,179 @@
+import xml.etree.ElementTree
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+import tiledb
+from tiledb.tests.common import assert_captured, DiskTestCase, has_pandas
+
+
+class AttributeTest(DiskTestCase):
```

Comment:
Would you mind to add a check for whether we can modify an attribute that is already in an ArraySchema? I think that should be blocked, but not 100% (it _will_ be blocked by libtiledb in the future)